### ISSUE TITLE
Meeting Minutes migrated to Members Area section on new site

### ISSUE THREAD

**Issue by rvizena** [UCSantaBarbara/webguide#414]
How should we do this? 
Some options to discuss: 

- Use the Book module and have chapters for each Year and each book page is a new month.
- Simple table with date and link to Google Doc.

*Find way to enable the whole group to view Minutes together (David suggestion) during a meeting.

**Comment by tenken** [comment]
@rvizena 
I offered a possible idea to land people onto the page of the most current meeting minutes agenda. Alluding to an Alias or something on login like `/members-area/meeting-minutes/current` for example to just drop logged in members to the current meeting minutes page.  I noted though, that if we use Drupal for meeting minutes we cannot collaboratively edit/create meeting minutes as we do on Google Docs.  On the old it.ucsb.edu group we simply assigned 1 person to take minutes every meeting on paper to create the on-site meeting minutes page. We could always just go that route on the site new site.
